geode-junit/src/main/java/org/apache/geode/test/micrometer/TimerAssert.java

```diff
@@ -49,6 +63,19 @@ public TimerAssert hasCount(Condition<? super Long> condition) {
     return myself;
   }
 
+  /**
+   * Verifies that the timer's total time satisfies the given condition.
+   *
+   * @param timeUnit the time unit to which to convert the total time before evaluating
+   * @param totalTime the expected value of the timer's total time
+   * @return this assertion object
+   * @throws AssertionError if the timer is {@code null}
+   * @throws AssertionError if the condition rejects the timer's total time
+   */
+  public TimerAssert hasTotalTime(TimeUnit timeUnit, double totalTime) {
```

Comment:
At some point (not now) we may want to add a tolerance parameter. Testing `double` for equality can cause trouble due to rounding.